target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4019-wpj419.dts

```diff
@@ -348,6 +348,16 @@
 			status = "okay";
 		};
 	};
+	
+	keys {
+		compatible = "gpio-keys";
+		
```

Comment:
This "empty" line should really be empty and not contain any tabs or other whitespaces. I've removed that myself, so no need to do anything now. Just to tell you for the next time you send a patch. :-)